src/ensembl/src/shared/helpers/urlHelper.ts

```diff
@@ -0,0 +1,4 @@
+export const speciesSelector = () => '/app/species-selector';
+export const customDownload = () => '/app/custom-download';
+export const browser = (genomeId: string, focus: string, location: string) =>
+  `/app/browser/${genomeId}?focus=${focus}&location=${location}`;
```

Comment:
This can’t be right. It should be possible to generate an "empty" genome browser url (`/app/browser`) that is going to show a screen with example objects for the first selected species.